### PR TITLE
Add PyTorch

### PR DESCRIPTION
**Pull request by sigseg5**

### CODE REVIEW

File: projects/pytorch/dump_sydr.cc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+#include <string>
+#include <sstream>
+
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/mobile/module.h>
+#include <torch/csrc/jit/serialization/import.h>
+#include <torch/csrc/jit/runtime/instruction.h>
+#include <c10/util/Flags.h>
+
+#include <fstream>
```
Comment (apach301) [inline projects/pytorch/dump_sydr.cc:27]: Not need in this

File: projects/pytorch/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+`sydr-fuzz -c load.toml run`
+
+* mobile_fuzz
+
+`sydr-fuzz -c mobile.toml run`
+
+## Applyed patches
+
+* miniz.* – Updated miniz version for fix segmentation fault.
+* stoull.patch – Added additional caffe2 version check for deeper fuzzing.
```
Comment (apach301) [inline projects/pytorch/README.md:40]: miniz.* – Updated miniz version to fix segmentation fault.
stoull.patch – Catch stoull exception to allow the fuzzer go deeper

File: projects/pytorch/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+FROM sweetvishnya/ubuntu20.04-sydr-fuzz
+
+MAINTAINER Kirill Parhomenko
+
+# Install build dependencies.
+RUN pip3 install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions 
+
+# set timezone
+RUN ln -snf /usr/share/zoneinfo/Europe/Moscow /etc/localtime
```
Comment (SweetVishnya) [inline projects/pytorch/Dockerfile:25]: Timezone could be set via docker `-v /etc/localtime:/etc/localtime:ro`:

```
sudo docker run --privileged --network host -v /etc/localtime:/etc/localtime:ro --rm -it -v $PWD:/fuzz oss-sydr-fuzz-pytorch /bin/bash
```

File: projects/pytorch/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# PyTorch
+
+PyTorch is an open source machine learning framework based on the Torch library, used for applications such as computer vision and natural language processing.
+
+## Perfomance note
+
+This project uses some perfomance related settings and you can tune this for your machine:
+
+* `MAX_JOBS=100` in build_*.sh – The maximum number of jobs to build the whole project, if your machine has fewer threads, the build system will use all the threads.
```
Comment (SweetVishnya) [inline projects/pytorch/README.md:9]: Why don't we use nproc?

File: projects/pytorch/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+* `-rss_limit_mb=30720` in *.toml - Memory usage limit for libFuzzer (in Mb), default 30GB. Use 0 to disable the limit. If an input requires more than this amount of RSS memory to execute, the process is treated as a failure case. The limit is checked in a separate thread every second.
+
+## Build Docker
+
+`docker build -t oss-sydr-fuzz-pytorch .`
```
Comment (SweetVishnya) [inline projects/pytorch/README.md:15]: dollar signs before commands, you may check other READMEs

File: projects/pytorch/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+`sydr-fuzz -c dump.toml run`
+
+* load_fuzz
+
+`sydr-fuzz -c load.toml run`
+
+* mobile_fuzz
+
+`sydr-fuzz -c mobile.toml run`
+
```
Comment (SweetVishnya) [inline projects/pytorch/README.md:44]: cmin, security, casr docs

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/pytorch/README.md (hunk 4 above)
Comment (sigseg5) [reply projects/pytorch/README.md:9]: Done

Comment (SweetVishnya) [inline projects/pytorch/README.md:9]: obsolete comment

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/pytorch/README.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+Run hybrid fuzzing:
+
+    `$ sydr-fuzz -c mobile.toml run`
+
+## Security predicates
+
+    `$ sydr-fuzz -c <target_name>.toml security`
+
+## Crash amalysis with Casr
```
Comment (SweetVishnya) [inline projects/pytorch/README.md:65]: analysis

File: projects/pytorch/README.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+    `$ sydr-fuzz -c <target_name>.toml casr`
+
+## Applyed patches
```
Comment (SweetVishnya) [inline projects/pytorch/README.md:69]: applied

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**